force-app/main/default/classes/ServiceParticipantRollupsService_TEST.cls

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ *  * Copyright (c) 2022, salesforce.com, inc.
+ *  * All rights reserved.
+ *  * SPDX-License-Identifier: BSD-3-Clause
+ *  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ *
+ */
+@IsTest
+public with sharing class ServiceParticipantRollupsService_TEST {
+    private static ServiceParticipantRollupsService rollupService = new ServiceParticipantRollupsService();
+
+    @TestSetup
+    private static void createTestData() {
+        // By turning off none, we turn on all
+        TestUtil.turnOffFeatureGates(new Set<String>{});
+        TestDataFactory.generateAttendanceData('Monthly');
+    }
+
+    @IsTest
+    private static void shouldUpdateParticipantsEnrolled() {
+        createTestData();
+        Integer expectedEnrolledParticipants = 2;
+        ServiceParticipantRollupsService rollupService = new ServiceParticipantRollupsService();
+
+        List<ServiceSchedule__c> schedulesToReturn = new List<ServiceSchedule__c>(
```

Comment:
Just for future reference, I think this data could be setup using the TestUtil.RecordBuilder framework and avoid having to insert or query any actual records (similar to `ServiceDeliveryService_TEST.createBuckets()`)